packages/manager/src/features/Databases/DatabaseCreate/DatabaseCreate.tsx

```diff
@@ -1,19 +1,426 @@
+import {
+  CreateDatabasePayload,
+  DatabaseType,
+  DatabaseVersion,
+  Engine,
+  FailoverCount,
+  ReplicationType,
+} from '@linode/api-v4/lib/databases/types';
+import { createDatabaseSchema } from '@linode/validation/lib/databases.schema';
+import { useFormik } from 'formik';
+import { groupBy } from 'ramda';
 import * as React from 'react';
-import useFlags from 'src/hooks/useFlags';
+import { useHistory } from 'react-router-dom';
+import MySQLIcon from 'src/assets/icons/mysql.svg';
+import { makeStyles, Theme } from 'src/components/core/styles';
+import Divider from 'src/components/core/Divider';
+import FormControl from 'src/components/core/FormControl';
+import FormControlLabel from 'src/components/core/FormControlLabel';
+import FormHelperText from 'src/components/core/FormHelperText';
+import Paper from 'src/components/core/Paper';
+import RadioGroup from 'src/components/core/RadioGroup';
+import Typography from 'src/components/core/Typography';
+import BreadCrumb from 'src/components/Breadcrumb';
+import Button from 'src/components/Button';
+import CircleProgress from 'src/components/CircleProgress';
+import Select, { Item } from 'src/components/EnhancedSelect/Select';
+import SingleValue from 'src/components/EnhancedSelect/components/SingleValue';
+import RegionSelect from 'src/components/EnhancedSelect/variants/RegionSelect';
+import RegionOption from 'src/components/EnhancedSelect/variants/RegionSelect/RegionOption';
+import ErrorState from 'src/components/ErrorState';
+import Grid from 'src/components/Grid';
+import MultipleIPInput from 'src/components/MultipleIPInput';
+import Notice from 'src/components/Notice';
+import Radio from 'src/components/Radio';
+import TextField from 'src/components/TextField';
+import SelectPlanPanel from 'src/features/linodes/LinodesCreate/SelectPlanPanel';
+import { typeLabelDetails } from 'src/features/linodes/presentation';
+import {
+  useDatabaseVersionsQuery,
+  useDatabaseTypesQuery,
+  useCreateDatabaseMutation,
+} from 'src/queries/databases';
+import { useRegionsQuery } from 'src/queries/regions';
+import { formatStorageUnits } from 'src/utilities/formatStorageUnits';
+import getSelectedOptionFromGroupedOptions from 'src/utilities/getSelectedOptionFromGroupedOptions';
+import { handleAPIErrors } from 'src/utilities/formikErrorUtils';
+import { validateIPs } from 'src/utilities/ipUtils';
+import scrollErrorIntoView from 'src/utilities/scrollErrorIntoView';
 
-// eslint-disable-next-line @typescript-eslint/no-empty-interface
-interface Props {}
+const useStyles = makeStyles((theme: Theme) => ({
+  formControlLabel: {
+    marginBottom: theme.spacing(),
+  },
+  btnCtn: {
+    display: 'flex',
+    justifyContent: 'end',
```

Comment:
Here's the fix for the Create Button alignment. Safari must not accept `end`

```suggestion
    justifyContent: 'flex-end',
```